content/blog/2020-05-googlesheets4-0-2-0/index.Rmarkdown.orig

```diff
@@ -0,0 +1,186 @@
+---
+title: googlesheets4 0.2.0
+author: Jenny Bryan
+date: '2020-05-10'
+slug: googlesheets4-0-2-0
+categories:
+  - package
+tags:
+  - googlesheets4
+  - google
+photo:
+  author: Stephane YAICH
+  url: https://unsplash.com/photos/ufxd6aU0w9U
+---
+
+<!-- index.Rmarkdown is generated from index.Rmarkdown.orig -->
+<!-- Please edit that file and run precompile.R -->
+
+```{r setup, include = FALSE}
+can_decrypt <- gargle:::secret_can_decrypt("googlesheets4")
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>",
+  eval = can_decrypt
+)
+options(tibble.print_min = 5L, tibble.print_max = 5L)
+```
+
+```{r eval = !can_decrypt, echo = FALSE, comment = NA}
+message("No token available. Code chunks will not be evaluated.")
+```
+
+```{r auth, include = FALSE}
+googlesheets4:::gs4_auth_docs()
+```
+
+We're ecstatic to announce the release of googlesheets4 0.2.0 on CRAN:
```

Comment:
We'd normally just link the site to its first usage, but I assume you want to make it more prominent than usual?

Reply:
Yes. And I also favour making it very obvious that such links lead to a pkgdown site. As opposed to just using the package name as link text. I suppose that is my personal habit.